src/clj/rems/api/schema.clj

```diff
@@ -134,6 +134,9 @@
   {:field/id schema-base/FieldId
    :field/type (s/enum :attachment :date :description :email :header :ip-address :label :multiselect :option :phone-number :text :texta :table)
    :field/title schema-base/LocalizedString
+   (s/optional-key :field/date-bound) (rjs/field
+                                         (s/enum :none :past :future)
```

Comment:
Should we skip `:none` and consider a missing `field/date-bound` to be the same as it? If the bound does not exist, it does not need the value?

This follows the logic we have for `field/privacy` and `field/visibility` though. But perhaps they are sort of wrong too.